layers/meta-balena-genericx86/recipes-kernel/linux/kernel-image-initramfs.bbappend

```diff
@@ -0,0 +1,9 @@
+do_install_append_genericx86-64-ext () {
+    if [ "${SIGN}" != "true" ]; then
+        return 0
+    fi
+
+    for type in ${KERNEL_IMAGETYPE}; do
```

Comment:
```
if [ -f "${DEPLOY_DIR_IMAGE}/${type}.sig" ]; then
    install -m 0644 ${DEPLOY_DIR_IMAGE}/${type}.sig ${D}/boot/${type}.sig
fi
```
Using the above we have no need of the `SIGN` variable.

Reply:
Also, why not move this to meta-balena? If the signature files are present, let's install them.

Reply:
Done in https://github.com/balena-os/meta-balena/pull/2405